src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <main>
+      <div class="card" data-qa="card">
+        <img
+          class="card__image"
+          alt="imac"
+          src="./images/imac.jpeg"
+        >
+        <section class="product">
+          <h2 class="product__name">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
+          <p class="product__code">
+            Код товара: 195434
+          </p>
+          <div class="row">
+            <section class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </section>
+            <p class="product_review">
+              Отзывов: 5
+            </p>
+          </div>
+          <div class="row row--2">
+            <p class="price">
+              Цена:
+            </p>
+            <p class="product_price">
+              69 999 грн
+            </p>
+          </div>
+
+          <button class="button" data-qa="hover">
+            <a href="#" class="button__link">
+              КУПИТЬ
```

Comment:
```suggestion
              Купить
```
+text-transform: uppercase